<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Doodles Clicker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="1" alignment="Qt::AlignBottom">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Yu Gothic UI Semilight</family>
        <pointsize>128</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QPushButton" name="secondUpgradeButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Upgrade 2 | [COST: 500] [REWARD: 10]</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QPushButton" name="thirdUpgradeButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Upgrade 3 | [COST: 2500] [REWARD: 20]</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="gameLoopButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Game Loop</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="firstUpgradeButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Upgrade 1 | [COST: 10] [REWARD: 5]</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" alignment="Qt::AlignTop">
     <widget class="QLabel" name="incrementLabel">
      <property name="font">
       <font>
        <family>Yu Gothic UI Semilight</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>+ 1</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QPushButton" name="finalUpgradeButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Upgrade Final | [COST: 25000] [REWARD: 50]</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
